core/variant/variant_utility.cpp

```diff
@@ -829,22 +838,18 @@ static _FORCE_INLINE_ Variant::Type get_ret_type_helper(void (*p_func)(P...)) {
 		static void call(Variant *r_ret, const Variant **p_args, int p_argcount, Callable::CallError &r_error) { \
 			call_helperr(VariantUtilityFunctions::m_func, r_ret, p_args, r_error);                               \
 		}                                                                                                        \
-                                                                                                                 \
```

Comment:
I know this goes against the usual policy but in such macro code, I figure it might be best to keep things compat and avoid empty lines with a trailing `\`. The alternative would have been to add the missing separation lines + trailing `\` but it didn't feel *right*.